### PR TITLE
BTA-4755-update-xof-cash-details

### PR DESCRIPTION
**Pull request by linniethepooh**
Include additional optional XOF cash payout details in TZO documentation

Screenshot 2021-01-20 at 12 13 30

### CODE REVIEW

**Comment by mchochowski** [comment]
Should this also mention "wari" as accepted (and default) value for cash_provider param?

File: _docs/payout-details.md [hunk 1]
```diff
@@ -384,7 +384,12 @@ The payment reference can also be provided in the recipient details hash optiona
 
 ## XOF::Cash
 
-For Senegalese cash pickup requests please use:
+For Senegalese cash pickup requests there are two payout provider options available:
+
+1. Wari (default)
+2. Wizall
+
+For Wari cash pickup requests please use:
```
Comment (sztupy) [inline _docs/payout-details.md:392]: we could add the following here: `"cash_provider": "wari" // Optional`

File: _docs/payout-details.md [hunk 2]
```diff
@@ -398,6 +403,23 @@ For Senegalese cash pickup requests please use:
...
+  "identity_card_id": "2231324232", // Mandatory for Wizall
+  "identity_card_type": "PP", // Mandatory for Wizall; Values: "PP": Passport, "NI": National ID
+  "cash_provider": "wizall" // Mandatory for Wizall
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xof-cash-details" raw=data-raw %}
+
 Please note when sending `XOF::Cash` payments you should subscribe to the `recipient.pending` webhook, as that will broadcast the payment reference ID the customer need to use to obtain the funds. Example webhook response excerpt:
```
Comment (sztupy) [inline _docs/payout-details.md:423]: Please update the section below as well as Wizall does not support us sending the payment reference, and also we won't send back the cash pickup code. This is only for Wari

**Comment by linniethepooh** [comment]
@mchochowski i've added more detail. I've not listed the cash provider param under wari cash pick ups as it's not required but i have noted it's the default provider

**Review by sztupy — APPROVED** [review]
👍  nice

**Pull request merged**